### DISCUSSION TITLE
Lazy loading?

### DISCUSSION THREAD

**Discussion by jcdick1** [jellyfin/jellyfin-vue#272] [UI & UX]
I wasn't aware of these "discussion" areas of Github.  This is good.

I don't know if it requires server-side code or not, but I was wondering if the vue client UI will - or does already - have  "lazy loading" to allow scrolling the library, rather than paging through?

**Comment by heyhippari** [comment]
The discussion tab is a beta Github feature which we asked to be a part of :smiley: 

For lazy loading, we already do have an "infinite scroller" on this client. The entire library will be loaded, but only a couple of items will be shown, then updated when scrolling.

We don't have old-web style pagination (yet, it could probably be an option since I know some people prefer it for some reason).

We do have some performance issues with the virtual scroller at the moment, though, which I know @ferferga is looking into.

**Comment by ferferga** [comment]
I've just pushed a PR with some lazy loading goodness from the old client, although only for images for now: #275
